### PR TITLE
E2-2023: Fixing an issue with initialization

### PR DESCRIPTION
**Pull request by milos1290**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [E2-2023](https://leanplum.atlassian.net/browse/E2-2023)
People Involved   | @milos1290

### CODE REVIEW

**Comment by shivanshukumar** [comment]
Milos, please fix the PR name.

**Comment by shivanshukumar** [comment]
I am going to fix the PR name for now, but please make sure to name the PRs properly Thanks.

File: LeanplumSample/Assets/LeanplumSDK/LeanplumAndroid/LeanplumAndroid.cs [hunk 1]
```diff
@@ -268,6 +265,10 @@ public override void DisableLocationCollection()
             Leanplum.StartHandler startResponseAction)
         {
             Started += startResponseAction;
+
+            // This also constructs LeanplumUnityHelper and the game object.
+            gameObjectName = LeanplumUnityHelper.Instance.gameObject.name;
```
Comment (hrishileanplum) [inline LeanplumSample/Assets/LeanplumSDK/LeanplumAndroid/LeanplumAndroid.cs]: Is this change specific to Unity?

Comment (milos1290) [reply LeanplumSample/Assets/LeanplumSDK/LeanplumAndroid/LeanplumAndroid.cs]: yeah

**Pull request merged**